cogs/assistance-cmds/port0.vwii.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Port 0 on the vWii
+help-desc: Picture of port 0 for the vWii.
+image-url: https://cdn.discordapp.com/attachments/206934458954153984/1095031205881204746/wiiu_usb_ports.jpg
```

Comment:
Does this URL actually work with the new Discord updates?

Reply:
it does now